rootfs/api/models/release.py

```diff
@@ -201,6 +206,18 @@ def get_registry_auth(self):
 
         return auth
 
+    def get_registry_ecr_auth(self):
+        ecr_client = boto3.client('ecr', region_name='us-east-1')
```

Comment:
I'm not sure we currently support any other region other than us-east-1 for ECR but now that ECR is multi-region is this going to break anything? @kingdonb @mrferos @felixbuenemann 

If someone sets ECR that us-west-1 this could potentially break I think, no?

Reply:
I was worried about this line as well, how does a user configure a different region exactly... (it needs to be documented)

Reply:
Looks like we do support different regions for ECR. It's actually the object storage that we only support us-east-1, I must be getting confused.

Here: https://docs.teamhephy.com/installing-workflow/configuring-registry/#ecr